app/views/loan_documents/_form.slim

```diff
@@ -0,0 +1,14 @@
+= form_for @loan_document do |f|
```

Comment:
@ericksonmc I'm not understanding this form at all. In the controller, the new action was instanced with Document.new as the form object, which also has a File attribute, sure, the template file, but not the one to be assigned here.

And also, where is the loan ID and the Document ID references assigned? They weren't assigned at controller level, not here. I don't see how this feature can possibly work.